cypress/integration/accessibility_Component Overview_spec.js

```diff
@@ -0,0 +1,10 @@
+describe('Component overview check', () => {
+  beforeEach(() => {
+    cy.visit('/component_overview');
+    cy.injectAxe();
+  });
+
+  it.skip('Tests accessibility on the Component overview page', () => {
```

Comment:
Looks like the test is still being skipped?

Reply:
Forgot the push

Reply:
DONE

Reply:
Forgot the skip...